x-pack/plugins/security_solution/server/lib/risk_engine/utils/create_index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { ElasticsearchClient, Logger } from '@kbn/core/server';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import type { IndicesCreateRequest } from '@elastic/elasticsearch/lib/api/types';
+
+export const createIndex = async ({
+  esClient,
+  logger,
+  options,
+}: {
```

Comment:
It looks like the implicit return value here is `Promise<true | IndicesCreateResponse>`; is that correct?

Reply:
Nit: if we're not currently using the return value, we should just explicitly return `void` all the time.